qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -1030,7 +1033,28 @@ def _build_qeom_result(
 
 
 class QEOMResult(EigensolverResult):
-    """The results class for the QEOM algorithm."""
+    """The results class for the qEOM algorithm.
+
+    For more details about the definitions, please refer to https://arxiv.org/abs/1910.12890.
+
```

Comment:
We could add a sentence here stating that the following attributes exist on this class. But I think it would also be fine as is.